tests/unit/src/components/__snapshots__/VBreadcrumbs.test.js.snap

```diff
@@ -0,0 +1,109 @@
+
+      </span>
+    </li>
+  </ol>
+</nav>
+`;
+
+exports[`VBreadcrumbs Created Renders 3 crumbs 1`] = `
+<nav role="navigation" class="vts-breadcrumbs">
+  <ol itemscope="itemscope" itemtype="http://schema.org/BreadcrumbList" class="vts-breadcrumbs__list">
+    <li itemprop="itemListElement" itemscope="itemscope" itemtype="http://schema.org/ListItem" class="vts-breadcrumbs__item">
+      <a id="breadcumb-first">
+        First
+      </a>
+    </li>
+    <li itemprop="itemListElement" itemscope="itemscope" itemtype="http://schema.org/ListItem" class="vts-breadcrumbs__item">
+      <a id="breadcumb-second">
+        Second
+      </a>
+    </li>
+    <li itemprop="itemListElement" itemscope="itemscope" itemtype="http://schema.org/ListItem" class="vts-breadcrumbs__item">
+      <span class="active">
+        Third
+      </span>
+    </li>
+  </ol>
+</nav>
+`;
+
+exports[`VBreadcrumbs Created Renders correctly without props 1`] = `
+<nav role="navigation" class="vts-breadcrumbs">
+  <ol itemscope="itemscope" itemtype="http://schema.org/BreadcrumbList" class="vts-breadcrumbs__list"></ol>
+</nav>
+`;
+
+exports[`VBreadcrumbs Crumbs from routes Dashboard defaults to home text 1`] = `
+<li itemprop="itemListElement" itemscope="itemscope" itemtype="http://schema.org/ListItem" class="vts-breadcrumbs__item">
+  <span class="active">
+    Home
+  </span>
+</li>
+`;
+
+exports[`VBreadcrumbs Crumbs from routes Get text From meta title 1`] = `
+<nav role="navigation" class="vts-breadcrumbs">
+  <ol itemscope="itemscope" itemtype="http://schema.org/BreadcrumbList" class="vts-breadcrumbs__list">
+    <li itemprop="itemListElement" itemscope="itemscope" itemtype="http://schema.org/ListItem" class="vts-breadcrumbs__item">
+      <span class="active">
+        Title
+      </span>
+    </li>
+  </ol>
+</nav>
+`;
+
+exports[`VBreadcrumbs Crumbs from routes Get text From name 1`] = `
+<nav role="navigation" class="vts-breadcrumbs">
+  <ol itemscope="itemscope" itemtype="http://schema.org/BreadcrumbList" class="vts-breadcrumbs__list">
+    <li itemprop="itemListElement" itemscope="itemscope" itemtype="http://schema.org/ListItem" class="vts-breadcrumbs__item">
+      <span class="active">
+        Name
+      </span>
+    </li>
+  </ol>
+</nav>
+`;
+
+exports[`VBreadcrumbs Crumbs from routes Get text From path 1`] = `
+<nav role="navigation" class="vts-breadcrumbs">
+  <ol itemscope="itemscope" itemtype="http://schema.org/BreadcrumbList" class="vts-breadcrumbs__list">
+    <li itemprop="itemListElement" itemscope="itemscope" itemtype="http://schema.org/ListItem" class="vts-breadcrumbs__item">
+      <span class="active">
+        records/
+      </span>
+    </li>
+  </ol>
+</nav>
+`;
+
+exports[`VBreadcrumbs Crumbs from routes Prevent duplicate child routes (with trailing slash) 1`] = `
+<nav role="navigation" class="vts-breadcrumbs">
+  <ol itemscope="itemscope" itemtype="http://schema.org/BreadcrumbList" class="vts-breadcrumbs__list"></ol>
+</nav>
```

Comment:
Not sure if this snapshot or the next properly capture the expected outcome of the scenario. Wasn't sure what the point of the path/parent.path comparison was doing/for.